install.md

```diff
@@ -0,0 +1,30 @@
+# Paranomax Mobile App
+
+## Installation steps for Windows
+
+### Development environment
+Choose one of the environments below to write/maintain the code:
+- [Git](https://git-scm.com/downloads)
+- [GitHub CLI](https://cli.github.com/)
+- [GitHub desktop](https://desktop.github.com/download/)
+- [GitHub online](https://github.com/)
+
+### Link GitHub desktop with GitLab  
```

Comment:
same here, not sure this is needed, is it?

Reply:
True is a listing of different possibilities.